charts/rd-location-ref-data-load/Chart.yaml

```diff
@@ -3,7 +3,7 @@ appVersion: "1.0"
 description: A Helm chart for RD Location Ref Data Load App
 name: rd-location-ref-data-load
 home: https://github.com/hmcts/rd-location-ref-data-load
-version: 0.0.20
+version: 0.0.6
```

Comment:
Is version 0.0.6 not a downgrade of version 0.0.20. I can see version 0.0.6 was used on 23/10/2020 revision: cec92a6bc86a647151c49ec6d9db7de05c76aa89

Reply:
Ad discussed, revert this file since the version doesn't need to be changed if there are no other changes to the file

Reply:
Build was failed due to that I downgraded the version (https://build.platform.hmcts.net/blue/organizations/jenkins/HMCTS_j_to_z%2Frd-location-ref-data-load/detail/PR-349/3/pipeline/). 
Now I have reverted changes. Will check the build